### DISCUSSION TITLE
"OR" validation in form level

### DISCUSSION THREAD

**Discussion by abish373** [react-hook-form/react-hook-form#7983] [Q&A]
How to do **or** validation in RHF

Let's assume there is a three-input field. _First Name_, _Last Name_, _Full Name_. If the user enters _First Name_ and _Last Name_, the _Full Name_ field becomes optional. If users enter _Full Name_ then _First Name_ and _Last Name_ is optional.

[CSB](https://codesandbox.io/s/async-snow-r10gnk?file=/src/App.js)

**Comment by bluebill1049** [comment]
I think you will have to write a validate function to leverage `getValues()` API, and potentially trigger validation on any of those inputs.
